src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -761,12 +796,16 @@ def get_devices(_sys_block_path='/sys/block'):
 
     device_facts = {}
 
-    block_devs = get_block_devs_lsblk()
+    block_devs = get_block_devs_sysfs(_sys_block_path)
+
+    block_types = ['disk', 'mpath']
+    if os.environ.get("CEPH_VOLUME_USE_LOOP_DEVICES", False) is not False:
+        block_types.append('loop')
 
     for block in block_devs:
         devname = os.path.basename(block[0])
         diskname = block[1]
-        if block[2] not in ['disk', 'mpath']:
+        if block[2] not in block_types:
```

Comment:
`def get_block_devs_lsblk()` L734 won't be used anywhere else after that, should we drop it and all related tests in `tests/util/test_disk.py` altogether?